include/xref/list_xref.h

```diff
@@ -422,6 +422,9 @@ typedef enum _XRefDatabase {
     // Also, if XREF_COUNT > sizeof(uint16), enlarge struct OOVPA.XRefSaveIndex (and Value somehow)
 } XRefDatabase;
 
+// NOTE: XREF_PUBLIC_INDEX is only for library's internal API usage.
+#define XREF_PUBLIC_INDEX(xref_index) xref_index - XREF_KT_COUNT - 1
```

Comment:
The comment says that this macro should not be used by the client app. However, since the header is in the include directory, it is visible and clients can in fact see and use it. So, I suggest that the macro should be placed in a different header that's only visible internally by the library. If it doesn't exist, then create it.

Reply:
While it's true it is meant for internal library purpose. Although, I had it out here for possible users may want to use it to reduce array size of symbols that are inaccessible via library's APIs. Yet, they have to convert the index back to library's supplied index number.

i.e. unit test tool, but I think it was unnecessary for std map (I think) I used in unit test tool.

Reply:
If it has a valid public usage, then the comment is misleading and should be removed in that case.